A critical error has ocurred while running the game!
Error type: <class 'pygame.error'>
What happened: Unsupported image format
Where it happened:  file 'd:\Documentos\Programas\C\nundertale\main.py', at line 20Why it happned: Bad coding xD

Full error traceback:
 Traceback (most recent call last):
  File "d:\Documentos\Programas\C\nundertale\main.py", line 20, in main
    initial_lore_level = GameLevel(lore_sprites)
  File "d:\Documentos\Programas\C\nundertale\model\GameLevel.py", line 11, in __init__
    self.load_level_sprites()
  File "d:\Documentos\Programas\C\nundertale\model\GameLevel.py", line 19, in load_level_sprites
    self.sprite_list.append(GameSprite(raw, name))
  File "d:\Documentos\Programas\C\nundertale\model\GameSprite.py", line 7, in __init__
    self.loaded_image = image.load_extended(raw, name)
pygame.error: Unsupported image format
